ASP.NET Core/PdfViewerWebService_6.0/Pages/Error.cshtml

```diff
@@ -0,0 +1,26 @@
+﻿@page
```

Comment:
Hi @RanjithaAmirthalingam , Can you let me know why we include the new files in the sample? Is it necessary? 
We only need to update the package name and version.

Regards,

Parthipan

Reply:
For .net 6, created new application based on how to topic and replaced the files in the already created .net 6 application to make it as runnable.